packages/store/src/decorators/state.ts

```diff
@@ -45,3 +49,18 @@ export function State<T>(options: StoreOptions<T>) {
     stateClass[META_OPTIONS_KEY] = optionsWithInheritance;
   };
 }
+
+function ensureStateClassIsInjectable(target: StateClass): void {
```

Comment:
Can we move this to another file? Because in essence it is necessary for Ivy

`store/src/ivy/ensure-state-class-is-injectable.ts`

Reply:
OK.